aws-env.dev

```diff
@@ -0,0 +1,8 @@
+##this file store all evronment variables.
+PROJECT = lambda2sqs
+AWS_REGION = ap-southeast-1
+DEPLOY_S3_PREFIX = lambda2sqs
+AWS_PROFILE = uneet-dev
+DefaultSecurityGroup = sg-d8dde8a6
```

Comment:
@tuananh1508  can we use a consistent convention for the variable?
change `DefaultSecurityGroup` to `DEFAULT_SECURITY_GROUP`

Reply:
Updated..